pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1222,44 +1189,6 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListenerForPortOrUDS(l
 	}
 }
 
-func (configgen *ConfigGeneratorImpl) generateManagementListeners(node *model.Proxy, noneMode bool,
```

Comment:
Why was this removed ?

Reply:
Fixed: use the old when inbound listener doesn't capture all ports